### DISCUSSION TITLE
Uncaught TypeError: Object(...) is not a function at HTMLAnchorElement.eval

### DISCUSSION THREAD

**Discussion by JuanDavila1101** [nss-evening-cohort-14/evening-client-side#162] [Help Me!] [answered]
Juan Davila
I searched other tickets and found similar issues but I don't know if it applies.

![image](https://user-images.githubusercontent.com/76821980/110399932-51945200-803c-11eb-99e2-bb36c51c799b.png)

Googled links:
https://www.google.com/search?q=Uncaught+TypeError%3A+Object(...)+is+not+a+function+at+HTMLAnchorElement.eval&rlz=1C1CHBF_enUS812US820&oq=Uncaught+TypeError%3A+Object(...)+is+not+a+function+at+HTMLAnchorElement.eval&aqs=chrome..69i57.4068j0j7&sourceid=chrome&ie=UTF-8
https://stackoverflow.com/questions/4026891/javascript-uncaught-typeerror-object-is-not-a-function-associativity-question
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Not_a_function

links to my 
https://github.com/JuanDavila1101/ASSIGNMENT-Firebase-Pinterest/blob/main/src/javascripts/components/events/navigationEvents.js

![image](https://user-images.githubusercontent.com/76821980/110400541-68877400-803d-11eb-8500-cfa92a972334.png)

**Comment by JuanDavila1101** [comment, marked as answer]
I need to read through the warnings b/c the import of the non Default functions can show as warning but not throw an error until is needed, so fixed the issue by adding curly parenthesis to the exported function { getBoards }

**Reply by drteresavasquez** [reply]
Thank you!

**Reply by jimguistical** [reply]
Thank you, Juan!  I just had the same error and was missing brackets.
